### PR TITLE
Adding Datacom equipment

### PR DESCRIPTION
**Pull request by leoefellopes**

### CODE REVIEW

File: module-types/Datacom/LC-32GPON-DM4618.yaml
Comment (danner26) [inline module-types/Datacom/LC-32GPON-DM4618.yaml:1]: Generally the module port is one of the numbers, i.e. `1/2/1`, 2 might be the module slot. In cases like this, the interface name should be `1/{module}/1` which will automatically make it `2` if it is in module position `2`

You can see similar practices deployed in this file: https://github.com/netbox-community/devicetype-library/blob/master/module-types/Cisco/C9400-LC-48S.yaml

File: device-types/Datacom/DM4770-32CX.yaml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    type: 100gbase-x-qsfp28
+    description: 100G/40G Base-X (QSFP28)
+  - name: hundred-gigabit-ethernet 1/1/31
+    type: 100gbase-x-qsfp28
+    description: 100G/40G Base-X (QSFP28)
+  - name: hundred-gigabit-ethernet 1/1/32
+    type: 100gbase-x-qsfp28
+    description: 100G/40G Base-X (QSFP28)
+  - name: mgmt-1/1/1
+    type: 100base-tx
```
Comment (danner26) [inline device-types/Datacom/DM4770-32CX.yaml:117]: If this is a OOB management port, you should mark it as such with `mgmt_only: true`

File: device-types/Datacom/DM4380-12XS-plus-3CX.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    type: 100gbase-x-qsfp28
+    description: 100G/40G Base-X (QSFP28)
+  - name: hundred-gigabit-ethernet 1/1/2
+    type: 100gbase-x-qsfp28
+    description: 100G/40G Base-X (QSFP28)
+  - name: hundred-gigabit-ethernet 1/1/3
+    type: 100gbase-x-qsfp28
+    description: 100G/40G Base-X (QSFP28)
+  - name: mgmt-1/1/1
+    type: 100base-tx
```
Comment (danner26) [inline device-types/Datacom/DM4380-12XS-plus-3CX.yaml:30]: If this is a OOB management port, you should mark it as such with `mgmt_only: true`

File: device-types/Datacom/DM4370-4GT-plus-4GX-plus-4XS.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - name: gigabit-ethernet-1/1/5
+    type: 1000base-x-sfp
+  - name: gigabit-ethernet-1/1/6
+    type: 1000base-x-sfp
+  - name: gigabit-ethernet-1/1/7
+    type: 1000base-x-sfp
+  - name: gigabit-ethernet-1/1/8
+    type: 1000base-x-sfp
+  - name: mgmt-1/1/1
+    type: 100base-tx
```
Comment (danner26) [inline device-types/Datacom/DM4370-4GT-plus-4GX-plus-4XS.yaml:44]: If this is a OOB management port, you should mark it as such with `mgmt_only: true`

File: device-types/Datacom/DM4360-4GT-plus-4GX.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  - name: gigabit-ethernet-1/1/5
+    type: 1000base-x-sfp
+  - name: gigabit-ethernet-1/1/6
+    type: 1000base-x-sfp
+  - name: gigabit-ethernet-1/1/7
+    type: 1000base-x-sfp
+  - name: gigabit-ethernet-1/1/8
+    type: 1000base-x-sfp
+  - name: mgmt-1/1/1
+    type: 100base-tx
```
Comment (danner26) [inline device-types/Datacom/DM4360-4GT-plus-4GX.yaml:44]: If this is a OOB management port, you should mark it as such with `mgmt_only: true`

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by leoefellopes** [comment]
Made all requested changes

**Pull request merged**